lib/interpreters-lookup.coffee

```diff
@@ -58,6 +58,8 @@ module.exports =
     for p in atom.config.get('autocomplete-python.pythonPaths').split(';')
       for project in atom.project.getPaths()
         modified = p.replace(/\$PROJECT/i, project)
+        project_folder_name = project.match(/([^\/]*)\/*$/)[1]
```

Comment:
Should be something like `[_, ..., projectFolderName] = project.split(path.sep)` (and `require 'path'` somewhere if needed):
* Match coffeescript styleguide
* Correctly split path on windows where `\` is used
* Make it more obvious what we are doing here. Regex is hard to understand